### PR TITLE
fix: Checksum addresses

### PR DESCRIPTION
**Pull request by memoyil**
### Summary
🔄🛡️🧪

The pull request updates various contract and token addresses in the web app to use the checksum format, which adds a case-sensitive validation to Ethereum addresses. This improves the security, compatibility, and readability of the code and the features that rely on these addresses. It also fixes some minor typos and deletes an unused file. The pull request affects several test, config, state, hook, and view files, such as `helpers.test.ts`, `contracts.ts`, `utils.test.ts`, and `BuyModal/index.tsx`.

> _Sing, O Muse, of the valiant pull request_
> _That brought consistency to the code base_
> _And made the Ethereum addresses blessed_
> _With uppercase letters in every place._

### Walkthrough
*  Change the `id` field of the `betResponse` and `transformedBetResponse` objects to use uppercase letters for consistency with the checksum format of Ethereum addresses in the predictions helpers tests ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-314fd3a5a42ece3f38014f0d056a1eadf18d46d0189e01226530568e1cd1a68bL42-R42),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-314fd3a5a42ece3f38014f0d056a1eadf18d46d0189e01226530568e1cd1a68bL61-R61),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-314fd3a5a42ece3f38014f0d056a1eadf18d46d0189e01226530568e1cd1a68bL82-R82),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-314fd3a5a42ece3f38014f0d056a1eadf18d46d0189e01226530568e1cd1a68bL132-R132),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-314fd3a5a42ece3f38014f0d056a1eadf18d46d0189e01226530568e1cd1a68bL236-R236))
* Change the `address` field of the `address` object to use uppercase letters for consistency with the checksum format of Ethereum addresses in the address helpers tests ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-76d81eade974aa21422a54cfaba9ce7fd5e6450691b1353ef730ae169b1c769eL5-R5))
* Change the `0x0ed7e52944161450477ee417de9cd3a859b14fd0` key and argument to use uppercase letters for consistency with the checksum format of Ethereum addresses in the apr tests ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-32583130fde1a74b3d341c6faeb0d28040612f2b1109216a136078589451579fL15-R15),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-32583130fde1a74b3d341c6faeb0d28040612f2b1109216a136078589451579fL64-R68))
* Add a comment to disable the eslint rule that checks for the address type annotation in the utils tests, since the file does not use TypeScript ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-289b53064a90823553c8344982b04baf62eb586397d16917c0e3b3b924aef683R1))
* Change the `FACTORY_ADDRESS` and `ETH_FACTORY_ADDRESS` constants to use uppercase letters for consistency with the checksum format of Ethereum addresses in the farms tests ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-cd5633c6b0222127d5e612caff1cde2985e7803b384a500dc04b5cf53dd3b8aeL64-R64),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-cd5633c6b0222127d5e612caff1cde2985e7803b384a500dc04b5cf53dd3b8aeL75-R75))
* Change various contract addresses for different chain ids to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `contracts.ts` file ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07L17-R18),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07L108-R108),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-6ccab19ffbec70686dc2221f7b79432ece4302c36067c91063d258209c09ec07L133-R133))
* Change the `tokenAddress` and `account` fields of the `farmAuctions` object to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `farmAuctions.ts` file ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L26-R26),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L265-R265),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L355-R355),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L363-R363),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L371-R371),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L379-R379),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L395-R395),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L403-R403),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L411-R411),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L435-R435),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L443-R443),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L451-R451),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L483-R483),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L563-R563),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L643-R643),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L659-R659),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L675-R675),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L691-R691),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L699-R699),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-b7ced0b848dd3825fc91b9c6e23123a363d5d74e9492effe69a892525c332111L707-R707))
* Change the `address` field of the `ifo` object to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `ifo.ts` file ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-ed9f603aef08f97c4e697adb54f782c92560ff51571483b83175a849352580b8L61-R61),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-ed9f603aef08f97c4e697adb54f782c92560ff51571483b83175a849352580b8L86-R86),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-ed9f603aef08f97c4e697adb54f782c92560ff51571483b83175a849352580b8L156-R156),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-ed9f603aef08f97c4e697adb54f782c92560ff51571483b83175a849352580b8L293-R293))
* Change the `GELATO_NATIVE` constant to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `index.ts` file ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-e4c0879b3617130edb131741eba0816230cf6e87597ccfc6c98faf264ed9e50fL29-R29))
* Change the `TOKEN_BLACKLIST` and `ETH_TOKEN_BLACKLIST` arrays to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `info.ts` file ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-c7d554519824faf023f33d0f146e6078a5cbbaa378c21cf9d1ec53bf31d9035eL23-R52))
* Change the `id` field of the `DELIST_COLLECTIONS` object to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `delist.ts` file ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-ba9c784ac11dff78f9931bd56bfe10cdf7771df4a9f1ad0bc5a0a1fa1e1a0964L14-R14))
* Change the `address` field of the `baseToken` and `quoteToken` objects to use uppercase letters for consistency with the checksum format of Ethereum addresses in the v3 utils tests ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-e5bcd1071d802b0286ddee1b26fc608b6ec11eb26b8c9b472c32291030f280ebL8-R9),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-e5bcd1071d802b0286ddee1b26fc608b6ec11eb26b8c9b472c32291030f280ebL20-R21),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-e5bcd1071d802b0286ddee1b26fc608b6ec11eb26b8c9b472c32291030f280ebL41-R42))
* Change the `DAI_ADDRESS` constant and the `address` field of the `parseCallKey` and `toCallKey` functions to use uppercase letters for consistency with the checksum format of Ethereum addresses in the multicall actions tests ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-d327edf868683ceac0865326d66113ca1a16dd024bfe8cb1fa8a27fea4e4ff27L10-R10),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-d327edf868683ceac0865326d66113ca1a16dd024bfe8cb1fa8a27fea4e4ff27L19-R19),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-d327edf868683ceac0865326d66113ca1a16dd024bfe8cb1fa8a27fea4e4ff27L25-R25),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-d327edf868683ceac0865326d66113ca1a16dd024bfe8cb1fa8a27fea4e4ff27L38-R38),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-d327edf868683ceac0865326d66113ca1a16dd024bfe8cb1fa8a27fea4e4ff27L48-R48),[link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-d327edf868683ceac0865326d66113ca1a16dd024bfe8cb1fa8a27fea4e4ff27L54-R54))
* Change the `DAI_ADDRESS` constant to use uppercase letters for consistency with the checksum format of Ethereum addresses in the multicall reducer tests ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-78500918dec289fe8bbc973cab5f45e950751e099b8413595013601be9684450L11-R11))
* Change the `TESTNET_WBNB_NFT_ADDRESS` constant to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `BuyModal` component ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-8843f16de845398b01d77db06744f2c15ae28e4124991691122970113e7de248L38-R38))
* Delete the `constants.ts` file in the `Chart` component, since it is not used ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-d581f9cb518d534bfca398161e841103f058da69ed06765c76272bd62bf48535))
* Change the `ADMINS` array to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `constants.ts` file in the `V3Info` view ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-77f9b222e9f5de52041b06ae540e3a8cb32c1fd777c20be054eb88990cabc845L9-R29))
* Change the `TOKEN_HIDE` object to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `constants.ts` file in the `V3Info` view ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-77f9b222e9f5de52041b06ae540e3a8cb32c1fd777c20be054eb88990cabc845L35-R42))
* Change the `ADMINS` array to use uppercase letters for consistency with the checksum format of Ethereum addresses in the `config.ts` file in the `Voting` view ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-d510380eae6f7e0db69912c3ff3a8e0330ddb32f91d604abe65dc8a635057eb8L4-R4))
* Change the test description to use a space between "bet" and "response" for readability in the predictions helpers tests ([link](https://github.com/pancakeswap/pancake-frontend/pull/7944/files?diff=unified&w=0#diff-314fd3a5a42ece3f38014f0d056a1eadf18d46d0189e01226530568e1cd1a68bL148-R148))

### CODE REVIEW

File: apps/web/src/state/info/queries/tokens/tokenData.ts [hunk 1]
```diff
@@ -92,6 +92,7 @@ const fetchTokenData = async (
         twoWeeksAgo: ${TOKEN_AT_BLOCK(chainName, block14d, tokenAddresses)}
       }
     `
+    console.info(query)
```
Comment (chefilip) [inline apps/web/src/state/info/queries/tokens/tokenData.ts:95]: remove console.log?

File: apps/web/src/views/Info/components/InfoSearch/index.tsx [hunk 2]
```diff
@@ -324,8 +324,10 @@ const Search = () => {
                     <Flex>
                       <CurrencyLogo address={token.address} chainName={chainName} />
                       <Text ml="10px">
-                        <Text>{`${(token.address && subgraphTokenName[getAddress(token.address)]) || token.name} (${
-                          (token.address && subgraphTokenSymbol[getAddress(token.address)]) || token.symbol
+                        <Text>{`${
+                          (token.address && subgraphTokenName[isAddress(token.address) || undefined]) || token.name
```
Comment (chefilip) [inline apps/web/src/views/Info/components/InfoSearch/index.tsx:328]: Simplify this
```
subgraphTokenName[isAddress(token.address)] || token.name
```

**Review by chefilip — APPROVED** [review]

**Comment by chefilip** [comment]
lgtm

**Review by ChefMomota — APPROVED** [review]

**Comment by ChefJerry** [comment]
nice work 💯

**Pull request merged**